packages/path_provider/path_provider_macos/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# 2.0.2
+## 2.0.3
+
+* Fix link in README
```

Comment:
nit:
```suggestion
* Fix link in README.
```

Reply:
Fixed in all READMEs (and documented in a new style guide).